app/controllers/catalog_controller.rb

```diff
@@ -259,31 +259,29 @@ class CatalogController < ApplicationController
   def zip_images
     image_list = params[:image_list].split('||')
     itemid = params[:itemid]
+    temp_file = Tempfile.new("ddr-zip-#{Time.now.utc}")
+    zip_filename = itemid+'.zip'
```

Comment:
Preferred Ruby style is to use string interpolation instead of concatenation (See [https://github.com/bbatsov/ruby-style-guide#strings](https://github.com/bbatsov/ruby-style-guide#strings)): 

`zip_filename = "#{itemid}.zip"`